File: Sawtantar/Array&Matrix/SumArray.cpp
#include <iostream>
using namespace std;
#define MAX_SIZE 100

int array(int arr[], int start, int len);
int main()
{
   int arr[MAX_SIZE];
    int i,N,sum;
    cout<<"EnTer Limit of values to be enter: ";
    cin>>N;
    cout<<endl<<"Enter elements "<<endl;
    for(i=0;i<N;i++)
    {
    cin>>arr[i];
    }
    cout<<"SUM OF ARRRAY ELEMENTS ARE ";
    sum=array(arr,0,N);
    cout<<endl<<sum;
    return 0;
}
int array(int arr[], int start, int len)
{
    
    if(start>=len)
    return 0;
   return arr[start]+array(arr,start+1,len);
  
}

File: Arnav Sharma/Assignment-1/sum_array.java
package com.array;

import java.util.Scanner;

public class sum_array {
    public static void main(String[] args) {
        int n;
        Scanner sc = new Scanner(System.in);
        System.out.println("enter the no of elements");
        n = sc.nextInt();

        int[] arr = new  int[15];
        System.out.println("Enter the elements");
        for (int i=0; i<n; i++){
            arr[i] = sc.nextInt();
        }
        int sum = 0;
        for (int i = 0; i < arr.length; i++) {
            sum = sum + arr[i];
        }
        System.out.println("Sum of elements of an array: " + sum);
    }
}

